contribute.md

```diff
@@ -1,7 +1,60 @@
+## Contributing
+
+Hey there! We're thrilled that you'd like to contribute to this project. Your help is essential for keeping it great.
+
+Please note that this project is released with a [Contributor Code of Conduct](https://github.com/srinidh-007/HackotberFest2021). By participating in this project you agree to abide by its terms.
+
 # Rules & Regulations 
 
 -   This is Beginner Friendly, Kickstart for students and those who are new to opensource 
 -   Use any language.
 -   Don't spam it will be rejected immediately
 -   Python, C, C++, Java, HTML, CSS, JavaScript, Node, Data Structure and Algorithms, Android Projects.
 -   Anything valuable for Community
+
+## Issues and PRs
+
+This is a beginner friendly repository where all those people who are new to open source can contribute. The unique thing is you can use any
+language of your choice be it Python, C, C++, Java, HTML, CSS, JavaScript, Node, Data Structure and Algorithms, Android Projects which would be considered valuable to the open source community as a whole. Interested? See the #how to contribute.
```

Comment:
After java remaining are not languages, so just start a new line saying useful projects in web dev, data structures & Algo , Andriod are good place to start